src/locale/de.yml

```diff
@@ -170,7 +170,7 @@ feat:
         remove: Von Aufgabe entfernen
       add:
         alreadyAdded: Bereits hinzugefügt
-        placeholder: Telefonierer*in hinzufügen
+        placeholder: Tippe um Telefonierer*in zu suchen oder neu hinzuzufügen
```

Comment:
I can very confidently say, this is correctly translated since this is my native language :smiley: